controllers/projects.controller.js

```diff
@@ -71,11 +71,35 @@ const addingProject = catchAsync(async (req, res) => {
   logger.info(
     `the data ${req.body.ProjectName},${req.body.StartDate},${req.body.PiNumber} inserted successfully`
   )
-  if (addedData) {
+  if (
+    req.body.ProjectName !== '' &&
+    req.body.StartDate !== null &&
+    req.body.PiNumber !== null
```

Comment:
Let's refactor these validations. What if StartDate is undefined? Or not a date? It still passing the checks.